### ISSUE TITLE
Fix RPC example (again)

### ISSUE THREAD

**Issue by Thomasdezeeuw** [Thomasdezeeuw/heph#441]
Following pr #437 I believed the issue was fixed, but it came up again in pr #440.

```log
test src/actor_ref/rpc.rs - actor_ref::rpc (line 86) has been running for over 60 second
```

**Comment by Thomasdezeeuw** [comment]
I think this is fixed by the inbox update in #443, specifically pr https://github.com/Thomasdezeeuw/inbox/pull/10.

I'm removing it from the First Release milestone, but keeping it open in case it comes back.

**Comment by Thomasdezeeuw** [comment]
Haven't seen this anymore, so closing as fixed by #443 and https://github.com/Thomasdezeeuw/inbox/pull/10.
